components/filter/filter.js

```diff
@@ -150,6 +163,10 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 			const builtDimension = this._buildDimension(dimension);
 			return html`<d2l-menu-item text="${dimension.text}">
 				${builtDimension}
+				<div slot="supporting">
+					<span>${this._formatFilterCount(dimension.appliedCount, dimension.maxCount)}</span>
+					<d2l-offscreen>${this.localize('components.filter.filterCountDescription', { number: dimension.appliedCount })}</d2l-offscreen>
+				</div>
```

Comment:
This will eventually be swapped out for the new status indicator component, which I expect will take `text` and some type of `aria-label`/`description` for screenreaders.  Wanted to get the terminology of these fleshed out now.